epsagon/triggers/aws_lambda.py

```diff
@@ -439,7 +439,10 @@ def __init__(self, start_time, event, context):
         super(EventsLambdaTrigger, self).__init__(start_time)
 
         self.event_id = str(event['id'])
-        self.resource['name'] = str(event['resources'][0].split('/')[-1])
+        name = 'CloudWatch Events'
```

Comment:
in case of events bridge event, what would be the resource name for the event?

Reply:
the trigger resource name will be always 'CloudWatch Events' because EventBusName doesn't exist here.